pkg/controllers/cspc-controller/sync.go

```diff
@@ -438,11 +438,31 @@ func (pc *PoolConfig) syncCSPI(cspc *cstor.CStorPoolCluster) error {
 func (pc *PoolConfig) syncCSPIWithCSPC(cspc *cstor.CStorPoolCluster, cspi *cstor.CStorPoolInstance) error {
 	cspiCopy := cspi.DeepCopy()
 	klog.V(2).Infof("Syncing cspi %s from parent cspc %s", cspiCopy.Name, cspc.Name)
+
+	// Finding out the cspc pool spec in the following way:
+	// 1. Using node selectors i.e if node selector of cspc and cspi is
+	// matched then that is the spec of CSPC belongs to searching CSPI
+	//			(If above step failed then find using step2)
+	// 2. Find out using data raidgroup blockdevice names i.e if atleast
+	// one blockdevice name in CSPI data raid groups is matched to
+	// blockdevice name in data raidgroups in CSPC spec then
+
+	//NOTE: CSPC pools spec consist repetion of blockdevice names
```

Comment:
nit : repetion --> repetition ?

Reply:
Have ignored the nit_picks as of now... Will take care in a subsequent PR.